packages/contracts/contracts/identity/IdentityProxy.s

```diff
@@ -0,0 +1,200 @@
+pragma solidity ^0.4.24;
```

Comment:
For all the other contract files, we use the .sol extension.
Should we use that as well for IdentityProxy.s and ProxyFactory.s ?

Reply:
I had to name them .s so truffle doesn't recognize them and try to compile them. Not a great solution but the alternative was to to put them in a separate directory. I searched everywhere but unfortunately it's not possible to make truffle ignore certain contracts.